cypress/integration/spec.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ButtonSpecs } from './buttons';
+import { CheckboxesSpec } from './checkboxes';
+import { ColorSpec } from './color';
+import { ListsSpec } from './lists';
+import { TogglesSpec } from './toggles';
+
+// Organized this way to make one batch for all of the tests in Applitools,
+// otherwise it treats each file as a different batch and makes it hard to
+// see a single run as one unit.
+describe(`Clarity - ${Cypress.env('CLARITY_THEME')}`, () => {
+  // ButtonSpecs();
```

Comment:
This is commented out but xdescribed in the suite. Not sure what it should be. You may have covered this in dev share and I missed it.

Reply:
I just commented it out as well to be certain.